static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -59,11 +52,24 @@ const showJourneyAlert = (journeyEl: HTMLElement): void => {
             if (ev.isTrusted) {
                 ev.preventDefault();
                 Promise.all([
-                    shouldDisplayNewsletterWarning(journeyEl),
-                    shouldDisplayConsentWarning(journeyEl),
-                ]).then(([emptyNewsletters, emptyMarketing]) => {
-                    if (emptyNewsletters && emptyMarketing) {
-                        showModal('confirm-consents');
+                    missingNewsletters(journeyEl),
+                    missingConsents(journeyEl),
+                ]).then(([newslettersCheckboxInfo, marketingCheckboxInfo]) => {
+                    console.log(newslettersCheckboxInfo);
+                    if (
+                        newslettersCheckboxInfo.unchecked.length > 0 ||
+                        marketingCheckboxInfo.unchecked.length > 0
```

Comment:
Do we need these two lists of checkboxes separated if they are are just used to generate a combined list?

Reply:
See also [this](https://github.com/guardian/frontend/pull/19399/files#r176467305) As treating them equally is the final behaviour i'd merge the totals from `[newslettersCheckboxInfo, marketingCheckboxInfo]` in one array asap and forget they are separate in the rest of the code